atom/browser/atom_browser_context.h

```diff
@@ -77,6 +83,7 @@ class AtomBrowserContext : public brightray::BrowserContext {
   std::unique_ptr<AtomDownloadManagerDelegate> download_manager_delegate_;
   std::unique_ptr<WebViewManager> guest_manager_;
   std::unique_ptr<AtomPermissionManager> permission_manager_;
+  std::unique_ptr<SpecialStoragePolicy> storage_policy_;
```

Comment:
Should be `scoped_refptr` as `storage::SpecialStoragePolicy` is ref counted.

Reply:
@deepak1556 fixed